src/Sylius/Bundle/WebBundle/Resources/views/Frontend/Review/_list.html.twig

```diff
@@ -0,0 +1,16 @@
+{% for review in reviews %}
+    <div class="review">
+        <span class="pull-right">{{ review.createdAt|date }}</span>
+            <h4 class="title">
+                Title: {{ review.title }}
+            </h4>
+            Rating: {{ review.rating }}
+            <br>
+            Author: {{ review.author.email }}
```

Comment:
[RFC] I think many people would complain about their email being publicly available. We can use "John D." for "John Doe", it's quite anonymous. Any other ideas? @pjedrzejewski